src/shell_minimal.js

```diff
@@ -44,6 +44,10 @@ var ENVIRONMENT_IS_WEB = !ENVIRONMENT_IS_NODE && !ENVIRONMENT_IS_SHELL;
 #endif
 #endif
 
+#if USE_WASM_WORKERS
+var ENVIRONMENT_IS_WASM_WORKER = Module['$ww'];
```

Comment:
* We need a boolean bit of info to distinguish whether we are the main thread or the Worker so that we know how to route the different initialization based on this. (we could use `var ENVIRONMENT_IS_WASM_WORKER = typeof importScripts !== 'undefined';` here, but eyeing towards enabling possible `-s PROXY_TO_WORKER=1` use cases, this may be better - it's also smaller code size wise)